iothub/device/src/Messaging/Message.cs

```diff
@@ -185,25 +185,11 @@ public string ComponentName
             set => SystemProperties[MessageSystemPropertyNames.ComponentName] = value;
         }
 
-        /// <summary>
-        /// Unique string for identifying a received message when acknowledging it.
-        /// </summary>
-        internal string LockToken
```

Comment:
Since the complete "receive message - ack it" is now a single flow (as they have been combined), we don't need to hold on to the delivery tag or lock token anymore. Is that correct?

Reply:
Correct